src/modules/payment-backend-methods/hooks/afterRegistration.ts

```diff
@@ -0,0 +1,30 @@
+import VueAnalytics from 'vue-analytics'
```

Comment:
im not sure if we should use vueanalytycs here since they ar edepending on some other module